website/js/website.js

```diff
@@ -13,7 +13,7 @@ window.onload = function () {
     aboutCode = window.document.getElementById('context-about');
     aboutCode.innerText = context.aboutString();
 
-    canvas.controller.multiFrameNumber = 1024;
+    canvas.controller.multiFrameNumber = 8;
```

Comment:
is it important to change the multiFrameNumber? This seems like a big change from 1024 to 8.